doc/nrf/device_guides/working_with_nrf/nrf70/developing/raw_tx_operation.rst

```diff
@@ -0,0 +1,128 @@
+   * - data_rate
+     - Data rate at which packet is to be transmitted. Depends on ``tx_mode`` parameter. If ``tx_mode`` is set to legacy mode, data rate is the number provided. If ``tx_mode`` is set to HT/VHT/HE mode, the data rate is the MCS rate.
+   * - packet_length
+     - Packet length of 802.11 raw packet excluding the raw transmit packet header length.
+   * - tx_mode
+     - Mode describing if packet is VHT, HT, HE or Legacy.
+   * - queue
+     - Wi-Fi access category mapping for packet.
+   * - raw_tx_flag
+     - Flag indicating raw packet transmission. This is reserved for driver use.
+
+Refer structure ``nrfxlib/nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h/raw_tx_pkt_header`` for the raw packet header.
+
+Refer structure ``nrfxlib/nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h/nrf_wifi_fmac_ac`` for setting the ``queue`` parameter in the raw packet transmit header.
+
+Refer structure ``nrfxlib/nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h/nrf_wifi_fmac_rawtx_mode`` for setting the ``tx_mode``  parameter in the raw packet transmit header.
+
+.. _ug_nrf70_developing_raw_packet_transmit_operation:
+
+Raw packet transmit operation
+*****************************
+
+The raw packet data has to be encompassed in an 802.11 packet and prepended with the raw transmit header before it is transmitted to the nrf70 Wi-Fi device driver for transmission. The packet structure and raw packet operation is depicted below:
+
+.. figure:: images/nrf7000_packet_Injection_tx_Flow.svg
+   :alt: Raw packet transmit packet structure and raw packet operation flow
+
+   Raw packet transmit packet structure and raw packet operation flow
+
+.. _ug_nrf70_developing_channel_operation:
+
+Channel operation
+*****************
+
+Channel configuration for raw packet transmission during station idle mode is necessary. Though there is a fallback channel on which packets are transmitted when the raw packet channel is not configured, It is desirable to configure an explicit channel of operation for raw packets when station is in disconnected idle state. Raw packet transmissions will follow the below channel operation for raw packet transmission:
+
+* **Unconnected station + TX injection mode configured and channel is not set:** The raw packet is transmitted on channel 1.
+* **Unconnected station + TX injection mode configured and channel is set:** The raw packet is transmitted on configured channel.
+* **Station connected + TX injection mode configured:** The raw packet is transmitted on the present connected AP channel.
+* **Station disconnection from AP + TX injection mode configured (channel configured before connection to AP):** The packet is sent on configured channel for raw packet transmission.
+* **Station disconnection from AP + TX injection mode configured (channel not configured before connection to AP):** The packet is sent on channel 1 for raw packet transmission.
+
+.. _ug_nrf70_developing_salient_points:
+
+Salient points
```

Comment:
Fine with this section, but see that you don't really repeat things we already described.

Reply:
### okay